### ISSUE TITLE
Webhook subscriptions for same properties on different micro-services

### ISSUE THREAD

**Issue by cozzbie** [apaleo/api#285]
## Question

How can we have different services, subscribe to the same property for webhook notifications?

## Background

We have about 3 different services under one umbrella body and each one needs to have/manage its own subscriptions to apaleo for the same properties. We have noticed that sometimes when we try to subscribe to a property X from say service B, a subscription already exists from service A and thus the subscription request fails. So, we need to have a way, and we would to know if this already exists and how we can go about it, where, Service A and Service B can both have webhook subscriptions of the same Topic D to Property X.

Thanks

**Comment by pgrm** [comment]
hello @cozzbie the idea is that you subscribe once and manage the distribution of those messages on your side. If those services are really independent, then it's best practice to create separate client IDs for each service. If they are correlated, for instance different microservices working together, then you should implement a coordinator which forwards the messages to the correct micro services, either directly or via a pub-sub mechanism.

**Comment by cozzbie** [comment]
Thanks a lot.

**Comment by cozzbie** [comment]
Hi @pgrm, one more point of clarification. So I just confirmed that we do indeed have different client ids for our services so I'd like to ask if its possible that when we request all the subscriptions for say Service A, we see the subscriptions for Service B also? Both of different client ids.

**Comment by pgrm** [comment]
no, that's not possible, the filter by client is implicit and cannot be removed or changed

**Comment by cozzbie** [comment]
Thank you.
